py-rattler/src/error.rs

```diff
@@ -99,3 +104,4 @@ create_exception!(exceptions, IoException, PyException);
 create_exception!(exceptions, SolverException, PyException);
 create_exception!(exceptions, TransactionException, PyException);
 create_exception!(exceptions, LinkException, PyException);
+create_exception!(exceptions, ConvertSubdirException, PyException);
```

Comment:
`ConvertSubdirException` is missing both from the bindings as well as in the `error.py` file.